Documentation/WhenToUseBabylonNative.md

```diff
@@ -0,0 +1,170 @@
+motivated the development of Babylon React Native in the first place.
+
+While native graphics-related capabilities are certainly Babylon Native and 
+Babylon React Native's greatest strengths, they are by no means the only
+features for which Babylon Native is the right choice. Other native-only
+capabilities, advanced asynchrony, multi-JavaScript context... Because
+Babylon Native technologies provide access to and through the platform in
+ways that other platforms preclude, any usage that requires the app to go 
+beyond the "normal" set of allowed behaviors for Web-like applications is
+a good candidate for development in Babylon Native technologies. The more
+"normal" an app's scenario is, the less likely it needs or should use 
+Babylon Native's flexibility and power; but when "normal" isn't enough, 
+Babylon Native technologies will often be the best -- and sometimes the
+only -- viable choice.
+
+## The Native-First Perspective
+
+While Babylon Native technologies should largely be "last resorts" for 
+developers approaching the hybrid app space from a Web-first perspective,
+the opposite is true from a native-first perspective. From such a 
+perspective, the goal of a hybrid app is to enable Web-like rendering
+capabilities while incurring as few of the costs associated with Web
+technologies as possible; and at this goal, Babylon Native excels.
+Babylon Native is designed to make very few assumptions about the 
+environment into which it is integrated, which makes it possible to 
+"inject" Babylon Native into existing native applications which want
+to *use* Web-like rendering capabilities without pivoting to *become*
+Web-like hybrid apps. As an example, consider an academic codebase written
+in C (as so many of them are) for use in some spatially-inclined scientific
+field such as astronomy. Adding a visualizer to this existing codebase
+would be extremely difficult to do from scratch, and the resulting 
+visualizer would likely be very rudimentary in its appearance and 
+capabilities. However, a visualizer built on Babylon Native could very
+easily be encapsulated behind a C API and added to the existing codebase,
+providing the full rendering power of Babylon.js at a very, very minimal
+code cost.
+
+In short, in all the ways that Babylon Native technologies are "last 
+resort" choices from a Web-first perspective, Babylon Native should be
+among the first options considered when looking at adding rendering from 
+a native-first perspective. Of course, Babylon Native will not be as fast
+or powerful as a full-native gaming solution like Unity or Unreal, so
+native usages that require that kind of top-tier power will likely not be 
+able to make do with Babylon Native. However, for native apps looking to 
+quickly and easily add feature-rich rendering, few alternatives will 
+provide as much rendering capability for as little native infrastructure 
+cost as Babylon Native can offer.
```

Comment:
The arguments for using Babylon Native may be download size. Maybe we can incorporate this into the text?